src/components/sbb-form-field/sbb-form-field.scss

```diff
@@ -0,0 +1,172 @@
+}
+
+:host(.form-field--size-l) {
+  --form-field-border-radius: var(--sbb-border-radius-8x);
+  --form-field-height: var(--sbb-size-form-element-l-min-height);
+  --form-field-padding-inline: var(--sbb-spacing-responsive-xxxs);
+}
+
+:host(.sbb-invalid) {
+  --form-field-border-color: var(--sbb-color-red125-default);
+  --form-field-text-color: var(--sbb-color-red125-default);
+
+  @include ifForcedColors {
+    --form-field-border-color: LinkText;
+    --form-field-text-color: LinkText;
+  }
+}
+
+:host(.form-field--readonly),
+:host(.form-field--disabled) {
+  --form-field-background-color: var(--sbb-color-milk-default);
+  --form-field-border-color: var(--sbb-color-graphite-default);
+}
+
+:host(.form-field--readonly) {
+  --form-field-label-color: var(--sbb-color-granite-default);
+  --form-field-prefix-color: var(--sbb-color-granite-default);
+}
+
+:host(.form-field--disabled) {
+  --form-field-label-color: var(--sbb-color-graphite-default);
+  --form-field-prefix-color: var(--sbb-color-graphite-default);
+  --form-field-text-color: var(--sbb-color-graphite-default);
+
+  @include ifForcedColors {
+    --form-field-label-color: GrayText;
+    --form-field-prefix-color: GrayText;
+    --form-field-text-color: GrayText;
+  }
+}
+
+:host(.form-field--borderless) {
+  --form-field-border-color: transparent;
+}
+
+:host(.form-field--error-space-reserve) {
+  --form-field-error-min-height: calc(
+    var(--sbb-typo-line-height-body-text) * var(--sbb-font-size-text-xs)
+  );
+  --form-field-error-padding-block-start-override: var(--form-field-error-divider-width);
+}
+
+.form-field__space-wrapper {
+  display: flex;
+  flex-direction: column;
+  @include zeroWidthSpace;
+}
+
+.form-field__wrapper {
+  display: flex;
+  align-items: center;
+  gap: var(--form-field-gap);
+  padding-inline: calc(var(--form-field-padding-inline) - var(--sbb-border-width-1x));
+  border: var(--sbb-border-width-1x) solid var(--form-field-border-color);
+  border-radius: var(--form-field-border-radius);
+  height: var(--form-field-height);
+  background-color: var(--form-field-background-color);
+}
+
+// Ensure slotted inline elements are vertically centered
+::slotted(*[slot='prefix']),
+::slotted(*[slot='suffix']) {
+  display: flex;
+}
+
+::slotted(*[slot='prefix']) {
+  color: var(--form-field-prefix-color);
+}
+
+.form-field__input-container {
+  flex: 1 1 auto;
+
+  // Prevents overflowing parent
+  min-width: 0;
+}
+
+// Label
+.form-field__label {
+  @include text-xs--regular;
+
+  display: flex;
+  max-width: 100%;
+
+  // Moves label down and input up to meet positioning requirements
+  margin-block-end: var(--form-field-label-to-input-gap);
+  color: var(--form-field-label-color);
+  user-select: none;
+  pointer-events: none;
+}
+
+::slotted(*[slot='label']),
+slot[name='label'] > span {
+  @include ellipsis;
+}
+
+// Input
+/* stylelint-disable plugin/stylelint-bem-namics */
+.form-field__input ::slotted(:is(input, select, [id^='sbb-field-input-'])) {
```

Comment:
why is [id^='sbb-field-input-'] needed?  should it not be something like before ".sbb-field-input"?

Reply:
Hmm.. let's see: 
- I think the value is wrong, it should be `[id^='sbb-form-field-input-']`, which is the default assigned id if the content slotted as input has no id;
- I think that Federico's idea behind this was to include also custom components, so with a template like `<sbb-form-field><my-input/></sbb-form-field>`, the `<my-input/>` would take the style rules, but..
- ..but it works only if the projected custom component has no id https://github.com/lyne-design-system/lyne-components/blob/8334e24728903770acee1c2fae9d15a4aabed1c4/src/components/sbb-form-field/sbb-form-field.tsx#L123

Conclusion: IMO line could be changed to `.form-field__input ::slotted(:is(input, select)) {`
but what about custom components? could they be skipped for now?

Reply:
We could use something generic like `.sbb-form-field-input` which custom form components could use/declare. We could also defer this until we actually need it.

Reply:
For now I've used the css class name `sbb-form-field-element`. We can adapt this in the future, if this is too limiting.